src/features/game/expansion/components/resources/Pebble.tsx

```diff
@@ -0,0 +1,254 @@
+        </div>
+      );
+
+      setToast({
+        icon: smallStone,
+        content: `+${pebble.stone.amount}`,
+      });
+
+      await new Promise((res) => setTimeout(res, 2000));
+      setCollecting(false);
+    } catch (e: any) {
+      displayPopover(
+        <span className="text-xs text-white text-shadow">{e.message}</span>
+      );
+    }
+  };
+
+  const timeLeft = getTimeLeft(pebble.stone.minedAt, PEBBLE_RECOVERY_TIME);
+
+  return (
+    <div
+      className="relative z-10 w-full h-full"
+      onMouseEnter={handleMouseHoverPebble}
+      onMouseLeave={handleMouseLeavePebble}
+    >
+      {/* Unmined pebble which is strikeable */}
+      {!mined && (
+        <div
+          ref={containerRef}
+          className="group cursor-pointer w-full h-full"
+          onClick={strike}
+        >
+          <Spritesheet
+            className="group-hover:img-highlight pointer-events-none z-10"
+            style={{
+              position: "absolute",
+              left: "-28px",
+              top: "-22px",
+            }}
+            getInstance={(spritesheet) => {
+              sparkGif.current = spritesheet;
+            }}
+            image={sparkSheet}
+            widthFrame={96}
+            heightFrame={64}
+            fps={24}
+            steps={6}
+            direction={`forward`}
+            autoplay={false}
+            loop={true}
+            onLoopComplete={(spritesheet) => {
+              spritesheet.pause();
+            }}
+          />
+        </div>
+      )}
+
+      <Spritesheet
+        style={{
+          position: "absolute",
+          left: "2px",
+          top: "-28px",
+          opacity: collecting ? 1 : 0,
+          transition: "opacity 0.2s ease-in",
+          imageRendering: "pixelated",
+        }}
+        className="pointer-events-none z-20"
+        getInstance={(spritesheet) => {
+          minedGif.current = spritesheet;
+        }}
+        image={dropSheet}
+        widthFrame={160}
+        heightFrame={64}
+        fps={18}
+        steps={10}
+        direction={`forward`}
+        autoplay={false}
+        loop={false}
+        onLoopComplete={(spritesheet) => {
+          spritesheet.pause();
+        }}
+      />
+
+      {/* Mined Pebble  */}
+      {mined && (
+        <div className="absolute" style={{ top: "14px", left: "9px" }}>
+          <img src={minedPebble} className="pointer-events-none -z-10 " />
+        </div>
+      )}
+      {/* Health bar shown when striking */}
+      <div
+        className={classNames(
+          "absolute top-10 left-1 transition-opacity pointer-events-none",
+          {
+            "opacity-100": touchCount > 0,
+            "opacity-0": touchCount === 0,
+          }
+        )}
+      >
+        <HealthBar percentage={collecting ? 0 : 100 - (touchCount / 2) * 100} />
+      </div>
+      {/* Recovery time panel */}
+      {mined && (
+        <div
+          className="absolute"
+          style={{
+            top: "41px",
```

Comment:
Currently there is no visual feedback to a player that a pebble is in 'recovery mode'. This would be confusing for players. I would suggest adding some opacity to the image to make it appear inactive.

E.g.

<img width="61" alt="Screen Shot 2022-07-08 at 9 32 59 am" src="https://user-images.githubusercontent.com/11745561/177888361-f9bab182-c29a-45d4-b5eb-0b1a978dfc22.png">